### PR TITLE
Filemode incorrect for Tamil translations

### PR DESCRIPTION
**Pull request by gerhardol**
## Proposed changes

Filemode changes have to actively suppressed for the changed files.

crlf fixes to match .gitattributes
(git add --normalize fixed this)

## Test methodology 

Build

## Merge strategy

I agree that the maintainer squash merge this PR (if the commit message is clear).

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
-﻿{
```
Comment (RussKie) [inline global.json:1]: Is this change necessary?

File: scripts/RepoLayout.props [hunk 2]
```diff
@@ -9,7 +9,7 @@
     <!-- Set version for runtimeconfig.json, used by DotnetRuntimeBootstrapper for minimal version to support
```
Comment (RussKie) [inline scripts/RepoLayout.props:9]: Is this change necessary?

**Review by RussKie — COMMENTED** [review]
👍 for the xlf

File: global.json (hunk 1 above)
Comment (gerhardol) [reply global.json:1]: Crlf is not matching.gitattributes
